src/dbg/debugger.cpp

```diff
@@ -1079,6 +1098,8 @@ static void cbCreateProcess(CREATE_PROCESS_DEBUG_INFO* CreateProcessInfo)
             sprintf_s(command, "bp " fhex ",\"entry breakpoint\",ss", (duint)CreateProcessInfo->lpStartAddress);
             cmddirectexec(command);
         }
+
+        bTraceRecordEnabledDuringTrace = settingboolget("Engine", "TraceRecordEnabledDuringTrace");

```

Comment:
Oh that's actually quite a bug in that case...
On Tue, 5 Jul 2016 at 04:03, Torusrxxx <notifications@github.com> wrote:

> In src/dbg/debugger.cpp
> <https://github.com/x64dbg/x64dbg/pull/807#discussion_r69500217>:
>
> > @@ -1079,6 +1098,8 @@ static void cbCreateProcess(CREATE_PROCESS_DEBUG_INFO* CreateProcessInfo)
> >              sprintf_s(command, "bp " fhex ",\"entry breakpoint\",ss", (duint)CreateProcessInfo->lpStartAddress);
> >              cmddirectexec(command);
> >          }
> > +
> > +        bTraceRecordEnabledDuringTrace = settingboolget("Engine", "TraceRecordEnabledDuringTrace");
>
> coz DBG_SETTINGS_UPDATED is not triggered when the application initializes
> so the bTraceRecordEnabledDuringTrace is not initialized. This line of
> code initializes it.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/x64dbg/x64dbg/pull/807/files/aa8d0b5d534b52f039bcf1e4c98ef6245f9bf1ba#r69500217>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACWCmaYdqr2fKzWzNYJWA9qOE_DY_utZks5qSbtfgaJpZM4I_uk9>
> .
>